### ISSUE TITLE
[Bug] Multiple Dicom Source Issue

### ISSUE THREAD

**Issue by Sufian-Afridi** [OHIF/Viewers#4309]
### Describe the Bug

When setting up multiple Dicom DataSources, it only displays data from first Source.
Even the dummy dataSources already configured into OHIF image files are behaving the same way.
To verify this you can do the following:
1. Remove the first Dicom Source. (Dont remove the second)
2. Go to UI and copy a study name.
3. Add back the removed Dicom Source.
4. Go to UI and search for the Study. You wont find it there.

### Steps to Reproduce

1. Pull latest or any other image
2. Start the container.

### The current behavior

Only displays first dataSource.

### The expected behavior

Should display both dataSources

### OS

CentOs 8 Stream

### Node version

No node installed

### Browser

Chrome v126

**Comment by sedghi** [comment]
We have a merge data source that was added by this PR, take a look hopefully that answer your question

https://github.com/OHIF/Viewers/pull/3788

**Comment by sedghi** [comment]
Maybe you can help us creating documentation for it and create a PR for us

**Comment by Sufian-Afridi** [comment]
> We have a merge data source that was added by this PR, take a look hopefully that answer your question
I tried it but there is a sourceName issue. I have multiple dicomweb servers, according to my understanding sourceName should be different for each dicomweb server. For instance

Server 1:
sourceName = dicomweb-1

Server 2
sourceName= dicomweb

Server 3
sourceName= dicomweb-2

Now when I view UI, the data shown is from server 2 , because I named it dicomweb.
No other name is being accepted. Only the server with sourceName dicomweb is displaying.
